### ISSUE TITLE
Haven app prevents Android device screen from turning off

### ISSUE THREAD

**Issue by deviantollam** [guardianproject/phoneypot#346]
I'm noticing that when the Haven app is the foreground app on an Android device, this seems to prevent the display from turning off... even if the Android O/S is configured to turn off the screen after X minutes.

Why this is a problem, potentially...

1. Preview is Valuable - many users likely appreciate the live preview window in Haven when you enable monitoring.  When the monitoring is active, you can see what the camera is seeing, in order to verify that your viewing angle is good, etc.  When i tap "Activate" and watch the countdown start, I like using the preview for this reason.  But then if I want my device screen off (to be covert and for the security of the device being locked) i have to pick the phone back up and press the power button, which of course disturbs the phone and changes the camera viewing angle, etc.

2. Turning off the Screen Manually Moves the Phone - as described above, manually pressing the power button will naturally disturb the device.  Even if it doesn't move the phone enough to grossly change the camera's view... it WILL disturb the phone enough to cause an event trigger, typically.  That needlessly starts recording.

Best case scenario would be for Haven to not flag the Android O/S with "keep display alive" so that folk could start monitoring, see the camera view that they want, walk away as the countdown delay happens, then even see (perhaps from across the room or at a side angle) that the screen says "ACTIVE" and they know it's working... then the phone automatically turns off the display X minutes later, locking the device.

**Comment by zcar300** [comment]
I agree. Also the phone uses a lot more battery even when the brightness is turned down. 

I am trying to use mine while plugged into a solar panel but it runs the battery down way too fast.
